test-util/src/test/java/org/triplea/test/lib/MatchBuilderTest.java

```diff
@@ -0,0 +1,82 @@
+package org.triplea.test.lib;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.hamcrest.Matcher;
+import org.hamcrest.MatcherAssert;
+
+import org.junit.jupiter.api.Test;
+
+class MatchBuilderTest {
+
+
+  /**
+   * Example output using a matcher with a failure injected.:
+   * 
+   * <pre>
+   *       MatcherAssert.assertThat(500, numberIsEqualToExampleMatcher(3));
+   java.lang.AssertionError:
+   Expected: Test value should have been equal to: 3
+   but: 500
+   * </pre>
+   */
+  @Test
+  void exampleWithDefaultBehavior() {
+    MatcherAssert.assertThat(0, numberIsEqualToExampleMatcher(0));
+    MatcherAssert.assertThat(5, numberIsEqualToExampleMatcher(5));
+  }
+
+
+  private Matcher<Integer> numberIsEqualToExampleMatcher(final int expected) {
+    return MatchBuilder.<Integer>builder()
+        .description("Test value should have been equal to: " + expected)
+        .checkCondition(testValue -> testValue == expected)
+        .build();
+  }
+
+  @Test
+  void basicMMismatchCase() {
```

Comment:
Extra `M` between `basic` and `Mismatch`.

Reply:
good catch - fixed - f0ead6e